client/src/i18n/en/stock.json

```diff
@@ -16,6 +16,7 @@
     "DETAILS"         : "Details",
     "DOCUMENT"        : "Document",
     "DONATION"        : "Donation",
+    "EMPTY"           : "No stock found",
```

Comment:
Should this say something more specific?  Like "You do not have any stock in your depot" or something?